modules/serverless-logic-troubleshooting-workflows.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-troubleshooting-workflows_{context}"]
```

Comment:
I see this under `Creating and managing workflows` section, which is not good, this should be following a chapter after we deploy the workflow to cluster.

Reply:
I see this is resolved but I dot not any related updates, is that correct? @kaldesai